code/game/mecha/equipment/mecha_equipment.dm

```diff
@@ -141,3 +159,6 @@
 /obj/item/mecha_parts/mecha_equipment/proc/log_message(message)
 	if(chassis)
 		chassis.log_message("<i>[src]:</i> [message]")
+
+/obj/item/mecha_parts/mecha_equipment/proc/self_occupant_attack()
+	return
```

Comment:
Why add this proc here? Futureproofing?

Reply:
It was being called from mob.dm in mode(), but after testing with it and that bit taken out it did literally nothing so it's been removed now.